uint32 timeout
bool isQueued
---
int32 result
uint64 queuedCmdIndex
